.openshift-ci/jobs/push-images/push-images.sh

```diff
@@ -1,11 +1,64 @@
 #!/usr/bin/env bash
 set -eo pipefail
 
+push_images_to_repos() {
+    local -n local_image_repos=$1
+    local -n local_tags=$2
+    local image_name=$3
+    local osci_image=$4
+
+    for repo in "${local_image_repos[@]}"; do
+        registry_rw_login "$repo"
+
+        for tag in "${local_tags[@]}"; do
+            image="${repo}/${image_name}:${tag}"
+            echo "Pushing image ${image}"
+            oc image mirror "${osci_image}" "${image}"
+        done
+    done
+}
+
+push_builder_image() {
+    # shellcheck source=SCRIPTDIR/../../drivers/scripts/lib.sh
+    source "$PROJECT_DIR/.openshift-ci/drivers/scripts/lib.sh"
```

Comment:
[nit] Could we move this somewhere global?

Reply:
Done